src/app/models/form.server.model.ts

```diff
@@ -443,26 +443,16 @@ const compileFormModel = (db: Mongoose): IFormModel => {
   }
 
   // Transfer ownership of the form to another user
-  FormSchema.methods.transferOwner = async function (
-    currentOwner: IUserSchema,
-    newOwnerEmail: string,
-  ) {
-    // Verify that the new owner exists
-    const newOwner = await User.findOne({ email: newOwnerEmail })
-    if (newOwner == null) {
-      throw new Error(
-        `${newOwnerEmail} must have logged in once before being added as Owner`,
-      )
-    }
-
-    // Update form's admin to new owner's id
+  FormSchema.methods.transferOwner = async function (currentOwner, newOwner) {
+    // Update form's admin to new owner's id.
     this.admin = newOwner._id
 
-    // Remove new owner from perm list and include previous owner as an editor
-    this.permissionList = filter(this.permissionList, (item) => {
-      return item.email !== newOwnerEmail
-    })
+    // Remove new owner from perm list and include previous owner as an editor.
+    this.permissionList =
+      this.permissionList?.filter((item) => item.email !== newOwner.email) ?? []
```

Comment:
did we ever figure out if a default value would be provided by mongoose on read, if it wasn't present in the DB?

Reply:
yea, will automatically have an empty array: https://mongoosejs.com/docs/schematypes.html#arrays

Reply:
so this should be safe?
```suggestion
    this.permissionList =
      this.permissionList.filter((item) => item.email !== newOwner.email)
```

Reply:
TypeScript will complain, but that is because it is optional in the schema. Will probably make a `IFormDocument` type that has all the default optionals populated as required to avoid this then.

Reply:
I'll do the retyping in a separate PR to avoid polluting this even more